contracts/libs/crypto/ECDSA512.sol

```diff
@@ -1,8 +1,7 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.4;
+pragma solidity ^0.8.21;
 
-import {call512, uint512} from "../bn/U512.sol";
-import {U512} from "../bn/U512.sol";
+import {call512, uint512, U512} from "../bn/U512.sol";
```

Comment:
This was done on purpose because for some reason it doesn't work with Idea solidity plugin.

Reply:
The same in mocks, please.

Reply:
Probably better to leave a comment in the code.